stl/inc/utility

```diff
@@ -509,6 +509,15 @@ struct _MSVC_KNOWN_SEMANTICS tuple_element<_Index, tuple<_This, _Rest...>>
     : tuple_element<_Index - 1, tuple<_Rest...>> { // recursive tuple_element definition
 };
 
+template <size_t _Index, class... _Types>

```

Comment:
Bill: Can they be in <xutility> instead? I worry about creating missing symbol warnings.

Bill later: But to even name these the user has to be able to make a tuple so they clearly already have the right thing.

No change requested.

Reply:
> Bill: Can they be in `<xutility>` instead? I worry about creating missing symbol warnings.

I suppose they could? This admittedly feels icky. 

I'd prefer to change _`pair-like`_ to use unqualified `get` so it can accept user-defined tuplish types a la structured bindings. (It's silly that we wrote this complicated concept that could equivalently be `is_specialization_of<T, pair> || (is_specialization_of<T, tuple> && tuple_size_v<T> == 2)`.)

Reply:
I mean it is not like this is set ins stone *runs away screaming in backward compatability*

Reply:
I originally put these here with the rest of the "tuple interface to `tuple`" declarations, but you've all convinced me it makes more sense to put them `<xutility>` immediately before `_Pair_like` with a comment explaining that they must be visible to unqualified lookup for the occurrences of `std::get` in the definition of the concept.